src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ArchiveGoalTest.kt

```diff
@@ -100,7 +99,7 @@ class ArchiveGoalTest : IntegrationTestBase() {
             .hasActionedByDisplayName("Bernie User")
         }
 
-      val eventPropertiesCaptor = ArgumentCaptor.forClass(Map::class.java as Class<Map<String, String>>)
+      val eventPropertiesCaptor = createCaptor<Map<String, String>>()
```

Comment:
What a superstar ⭐ 
These code warnings have been annoying me for ages and I wasn't sure how to fix them. Nice one 😁 👍